js/ad_manager_vast.js

```diff
@@ -608,7 +608,7 @@ OO.Ads.manager(function(_, $) {
       };
 
       this.initVpaidAd(this._properties['adWidth'], this._properties['adHeight'], viewMode,
-                  this._properties['adDesiredBitrate'], environmentVariables, creativeData);
+                  this._properties['adDesiredBitrate'], creativeData, environmentVariables);
```

Comment:
yep I tested and its working. I modified the code to match the spec: http://www.iab.com/wp-content/uploads/2015/06/VPAID_2_0_Final_04-10-2012.pdf

initAd(width : Number, height : Number, viewMode : String, desiredBitrate : Number,
[creativeData : String], [environmentVars : String]) : void